### DISCUSSION TITLE
Getting very low quality images

### DISCUSSION THREAD

**Discussion by theridon** [JPhilipp/powerdalle#1] [Q&A]
I just installed this following the instructions, but no matter what settings I use in the local UI, I'm getting low quality images that seem to not even be from Dalle. I used gitbash to set it up, and I get the local UI to run just fine in my browser.

Example:
![2023-11-26-01-12-57-ce58f2da](https://github.com/JPhilipp/powerdalle/assets/85383195/038f3537-0d57-45aa-b8cc-ff86415f1809)

Same prompt on Bing's Dalle, much better quality:
![OIG pwKsdZa5VoljI3OkVLk8](https://github.com/JPhilipp/powerdalle/assets/85383195/9af131c6-8352-4f03-9df7-8e273eb1a5b2)

Any idea what is causing this? Thank you!

**Comment by JPhilipp** [comment]
You are raising a very good point -- sadly, the Bing Dall-E 3 style is very different from the ChatGPT Dall-E 3 style. The latter is similar to the API. It is often hyperrealistic to the point of a children's book drawing, whereas Bing Create has a coolness about it.

When first programming with the API I had thought that there's a chance the "Natural" setting is the one Bing uses, but it doesn't look like it.

You can try to get it more towards Bing by certain words in the prompt. If anyone has successful prompt example words, please share them here. I sometimes go with things like "muted colors" or "studio photography" but haven't yet found a good catch-all.

**Comment by theridon** [comment]
Thanks for the response, it seems like indeed it does use a different model. I am trying to use Dalle's previous bing model-it used to give incredibly realistic depictions of things like animals.

Apparently dalle purposely messes with your prompt when accessing it via API. I decided to do a test and use the modified prompt (the one that dalle forces me to use) on bing, and it still gives different results, so it seems like it is a different model.

I did however have moderate success with one image:
![2023-11-26-01-59-44-a3ccf61a](https://github.com/JPhilipp/powerdalle/assets/85383195/88b8acba-a61a-4e7e-8153-332a1b16caa2)



The bird in this picture is well done, realistic and very close to dalle "2.5"-like you said, maybe using natural style does at least help.  And yet aside from the bird the image is horrible. If anyone has any luck with prompting producing realistic and high quality images please do let me know.

**Comment by JPhilipp** [comment]
Right, it's a challenge, but with the right prompts and enough tries (something which Power Dall-E helps with, incidentally), you can get rather realistic photos. Here's an example of something I'm currently working on. This uses the default Vivid style. It's a short prompt including the words "low angle from below" and "city photo" at the end.

![photo](https://github.com/JPhilipp/powerdalle/assets/1754503/cd3ce418-0bfd-4e5b-a04a-f819ec4f9363)
